R/data.R

```diff
@@ -52,3 +52,68 @@
 #' @source Compiled by Alessandro Chiarenza. See item descriptions for details.
 "GTS2012"
 
+#' Early tetrapod data from the Paleobiology Database
+#'
+#' A dataset of tetrapod occurrences ranging from the Carboniferous through to
+#' the Early Triassic. Data from: \url{https://paleobiodb.org/}.
+#' Dataset includes a range of variables relating to identification, geography,
+#' environmental context, traits and more. Those relevant to common paleobiological
+#' analyses are described below, but more information (and information on other
+#' variables) can be found at \url{https://paleobiodb.org/data1.2/}.
+#'
+#' @format A data frame with 5270 rows and 73 variables:
+#' \describe{
+#'   \item{occurrence_no}{Reference number given to the particular occurrence in
+#'   the Paleobiology Database}
+#'   \item{collection_no}{Reference number given to the Paleobiology Database
+#'   collection (locality) that the occurrence belongs to}
+#'   \item{identified_name}{Taxon name as it appears in the original publication,
+#'   which may include expressions of uncertainty (e.g. "cf.", "aff.", "?") or
+#'   novelty (e.g. "n. gen.", "n. sp.")}
+#'   \item{identified_rank}{The taxonomic rank, or resolution, of the identified
+#'   name}
+#'   \item{accepted_name}{Taxon name once the identified name has passed through
+#'   the Paleobiology Database's internal taxonomy, which collapses synonyms,
+#'   amends binomials which have been altered (e.g. species moving to another
+#'   genus) and updates taxa which are no longer valid (e.g. _nomina dubia_)}
+#'   \item{accepted_rank}{The taxonomic rank, or resolution, of the accepted
+#'   name}
+#'   \item{early_interval}{The oldest (or only) time interval within which the
+#'   occurrence is thought to have been deposited}
+#'   \item{late_interval}{The youngest time interval within which the occurrence
+#'   is thought to have been deposited}
+#'   \item{max_ma, min_ma}{The age range given to the occurrence}
+#'   \item{phylum, class, order, family, genus}{The taxa (of decreasing taxonomic
+#'   level) which the occurrence is identified as belonging to}
+#'   \item{abund_value, abund_unit}{The number (and units) of fossils attributed
+#'   to the occurrence}
+#'   \item{lng, lat}{The modern-day longitude and latitude of the fossil locality}
+#'   \item{collection_name}{The name of the Paleobiology Database collection
+#'   which the occurrence belongs to, typically a spatio-temporally restricted
+#'   locality}
+#'   \item{cc}{The country (code) where the fossils were discovered}
+#'   \item{paleolng, paleolat}{The longitude and latitude at which the fossils
```

Comment:
Maybe we should dump these columns? Our package will generate palaeocoordinates for occurrence. I am also worried that we still don't know exactly how these palaeocoordinates are being generated (not that it really matters here)

Reply:
I wanted something as close to a raw download as possible, but I don't mind taking these out if you'd prefer that.